CSharp/Library/Microsoft.Bot.Builder.Azure/TableBotDataStore2.cs

```diff
@@ -50,9 +50,17 @@ namespace Microsoft.Bot.Builder.Azure
     /// <see cref="IBotDataStore{T}"/> Implementation using Azure Storage Table,
     /// </summary>
     /// <notes>
-    /// The original TableBotDataStore put all conversation data in 1 partition in 1 table which severely limits the scalability. 
-    /// This TableBotDataStore2 class changes to use multiple tables, one tableName="{BotId}{ChannelId}[Conversation|User]" 
-    /// It then uses the UserId and ConversationId as the PartitionKeys within those tables to achieve much higher scalability charateristics
+    /// The original TableBotDataStore put all conversation data in 1 partition in 1 table which severely limited the scalability of it. 
+    /// The TableBotDataStore2 implementaion uses multiple tables, and the UserId or ConversationId as the PartitionKey within those tables to 
+    /// achieve much higher scalability charateristics.
+    /// 
+    /// tableName="{BotId}{ChannelId}"
+    /// BotStoreType             | PartitionKey      | RowKey         |
+    /// ---------------------------------------------------------------
+    /// UserData                 | {UserId}          | "user"         |
+    /// ConverationData          | {ConversationId}  | "conversation" |
+    /// PrivateConverationData   | {ConversationId}  | {UserId}       |
+    /// ---------------------------------------------------------------
     /// </notes>
     public class TableBotDataStore2 : IBotDataStore<BotData>
```

Comment:
Definitely like the new set up. However, we might want to consider creating a TableBotData store where the function (BotData) => {PartitionKey, RowKey} is provided by the user (we provide a default implementation, but users can override the structure with this mapping.